plugins/notification-resources/src/components/EmployeeInbox.svelte

```diff
@@ -204,21 +226,29 @@
     {#if loading}
       <Loading />
     {:else}
-      {#each docs as item, i (item._id)}
-        <NotificationView
-          value={item}
-          selected={selected === i}
-          {viewlets}
-          on:keydown={onKeydown}
-          on:click={() => {
-            selected = i
-            changeSelected(selected)
-          }}
-        />
+      {#each filteredDocs as item, i (item._id)}
+        <div class="with-hover">
+          <NotificationView
+            value={item}
+            selected={false}
+            {viewlets}
+            on:keydown={onKeydown}
+            on:click={() => {
+              selected = i
+              changeSelected(selected)
+            }}
+            preview
+          />
+        </div>
       {/each}
     {/if}
   </Scroller>
 </div>
+{#if space !== undefined}
+  <div class="reference">
+    <AttachmentRefInput bind:loading {space} {_class} objectId={messageId} on:message={onMessage} />
```

Comment:
I suppose it should be extension to inbox from chunter module.

Reply:
I've made `DirectMessageInput` mixin in `chunter`, attached it to `DirectMessage` class and used it here. Is this the right way to do it? Or did you mean that we need to add mixin in `notification` plugin and then use it in `chunter` model?

Reply:
Better to not do it by mixin, but just a class inside model, so it could be multiple footer/header components we could contribute into employee inbox. Etc. Same for buttons to open new message/etc. Let's do it in next PR.